plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -252,11 +256,11 @@ export class MySQLInstrumentation extends InstrumentationBase<
       const thisPlugin = this;
       diag.debug('MySQLInstrumentation: patched mysql query');
 
-      return function query(
+      return (
```

Comment:
function and arrow functions in js are not identical. In arrow function `=>` the `this` parameter will now point to the instrumentation object and not to the `mysql` object on which the patched function is called.

You can read more about it [here](https://blog.bitsrc.io/arrow-functions-vs-regular-functions-in-javascript-458ccd863bc1)

I believe it is important to retain the `function` syntax to keep the instrumentation functioning correctly

Reply:
will be fixed. thanks!